Array/readme.md

```diff
@@ -26,6 +26,7 @@ To create an array, you need to known the length (or size) of the array in advan
 
  * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
  * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
+ * Repeating And Missing Number ----> [Java](/Code/Java/Repeating_And_Missing_Number.java)
  * Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp) 
```

Comment:
Edit line 30 instead of adding it at line 20 like:
```suggestion
 * Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp) | [Java](/Code/Java/Repeating_And_Missing_Number.java)
```